server/routes/support/handlers/offenderDetail.ts

```diff
@@ -18,7 +23,33 @@ export default class OffenderDetailRoutes {
       ? await this.communityService.getStaffDetailByStaffCode(probationPractitioner?.staff.code)
       : undefined
     const hdcStatus = _.head(await this.prisonerService.getHdcStatuses([prisonerDetail], user))
+    const licenceSummary = await this.licenceServer.getLatestLicenceByNomisIdsAndStatus([nomsId], [], user)
 
+    let licencecrn = 'Not found'
+    let crd = 'Not found'
+    let releaseDate = 'Not found'
+    let sentenceStartDate = 'Not found'
+    let sed = 'Not found'
+    let led = 'Not found'
+    let tussd = 'Not found'
+    let licensetused = 'Not found'
+
+    if (licenceSummary != null) {
+      const licence = await this.licenceServer.getLicence(licenceSummary.licenceId.toString(), user)
+
+      licencecrn = licence.crn
+      crd = licence.conditionalReleaseDate
+      releaseDate = licence.actualReleaseDate
+      sentenceStartDate = licence.sentenceStartDate
+      sed = licence.sentenceEndDate
+      led = licence.licenceExpiryDate
+      tussd = licence.topupSupervisionStartDate
+        ? moment(licence.topupSupervisionStartDate).format('DD MMM YYYY')
+        : 'Not found'
+      licensetused = licence.topupSupervisionExpiryDate
```

Comment:
I think an alternate approach here would mean that we don't have to double-up variable names, so there's no need for `tused` and `licencetused`.
Even if we were to keep it, it would probably be better to be called `licenceTused` to follow camelcase